camel-idea-plugin/src/test/resources/testData/JavaCamelRouteLineMarkerProviderTestData.java

```diff
@@ -1,38 +1,29 @@
-/**
+package org.apache.camel.idea.gutter; /**
  * Licensed to the Apache Software Foundation (ASF) under one or more
  * contributor license agreements.  See the NOTICE file distributed with
  * this work for additional information regarding copyright ownership.
  * The ASF licenses this file to You under the Apache License, Version 2.0
  * (the "License"); you may not use this file except in compliance with
  * the License.  You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
  * Unless required by applicable law or agreed to in writing, software
  * distributed under the License is distributed on an "AS IS" BASIS,
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 import org.apache.camel.builder.RouteBuilder;
 import org.apache.camel.main.Main;
 
-public final class JavaCamelRouteLineMarkerProviderTestData {
-
-    private JavaAddEndpointIntentionTestData() { }
-
-    public static void main(String[] args) {
-        Main main = new Main();
-        main.addRouteBuilder(new MyRouteBuilder());
-        main.run(args);
+public class JavaCamelRouteLineMarkerProviderTestData extends RouteBuilder {
```

Comment:
I think this PR has checkstyle errors, when you build the code it should report errors.